tests/integration/test_integration.py

```diff
@@ -0,0 +1,100 @@
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+# Copyright 2019 The celery-dyrygent Authors. All rights reserved.
+
+import re
+import uuid
+from collections import (
+    defaultdict,
+)
+
+import celery
+import pytest
+from app import (
+    order_task,
+    read_logs,
+    return_value_task,
+)
+
+from celery_dyrygent.workflows import (
+    Workflow,
+)
+
+
+@pytest.fixture(scope='function')
+def test_name():
+    return str(uuid.uuid4())
+
+
+class Helpers:
+    @staticmethod
+    def parse_logs(logs):
+        task_regex = r'app.(?:order|return_value)_task\[([\w-]+)\]: Test name: ([\w-]+) Test value: ([\w-]+)'
+        grouped_logs = defaultdict(list)
+
+        for log in logs.splitlines():
+            findings = re.search(task_regex, log)
+            if findings:
+                _, test_name, test_value = findings.groups()
```

Comment:
If you're going to throw away the first capture group, you might as well mark it as non-capturing with `(?:...)`.